### DISCUSSION TITLE
trusted-click-element

### DISCUSSION THREAD

**Discussion by dportvine** [uBlockOrigin/uAssets#20098] [Q&A] [answered]
`https://www.facebook.com/NASA/?locale=en`

`facebook.com##+js(trusted-click-element, [aria-label="Close"])` works for english language, how to make it work for different languages?

<details>
<summary>Screenshot(s)</summary>

![2023-10-15_221731](https://github.com/uBlockOrigin/uAssets/assets/111344219/ebe21747-80b6-47e1-9f40-da95c40f02c5)
</details>

**Comment by gorhill** [comment, marked as answer]
facebook.com##+js(trusted-click-element, [role="dialog"]:has(#login_popup_cta_form) [role="button"][aria-label])

Remove `:has()` for Firefox (might be more subject to false positive):

    facebook.com##+js(trusted-click-element, [role="dialog"] [role="button"][aria-label])

The close button appears to always be the first one.

**Reply by dportvine** [reply]
Tnx for your help & tnx for this scriplet.
